packages/experimental/Button/src/Button.styling.ts

```diff
@@ -88,3 +79,18 @@ export const getDefaultSize = (): ButtonSize => {
 
   return 'medium';
 };
+
+export const contentStyling = (tokens: ButtonTokens, theme: Theme, contentColor: ColorValue, fontStylesTokens: FontTokens) => {
```

Comment:
Can we pass in spacingIconContent instead of tokens as the arg? It just seems odd that we pass in contentColor and fontStylesTokens separately and not spacingIconContent.

Reply:
I passed color and tokens, because it's different in CompoundButton for content and secondaryContent

Reply:
I guess for me since the only thing you're taking from tokens is spacingIconContent it seems a bit much to pass in the entire tokens object. I don't think it really matters though, this can stay as is.